code/__HELPERS/unsorted.dm

```diff
@@ -2079,3 +2079,30 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 	))
 	query_accesslog.warn_execute()
 	qdel(query_accesslog)
+
+/proc/slot_bitfield_to_slot(input_slot_flags) // Kill off this garbage ASAP; slot flags and clothing flags should be IDENTICAL. GOSH DARN IT. Doesn't work with ears or pockets, either.
```

Comment:
Awful, just awful. 

we define slots with `slot_x` that are just numerical numbers....ideally these would all just be the same thing, but here we are---this is definitely one of those "please do this refactor, pleaseeee" situations.

Reply:
The real sin here is use of lowercase defines
```dm
// slots
#define slot_back 1
#define slot_wear_mask 2
#define slot_handcuffed 3
#define slot_l_hand 4
#define slot_r_hand 5
#define slot_belt 6
#define slot_wear_id 7
#define slot_l_ear 8
#define slot_glasses 9
#define slot_gloves 10
#define slot_head 11
#define slot_shoes 12
```